Sources/PitchSpeller/Wetherfield/EdgeCapacity.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  EdgeCapacity.swift
+//  NotationModel
+//
+//  Created by Ben Wetherfield on 06/16/18.
+//
+
+enum EdgeCapacity:
```

Comment:
Ahh, nice! I avoided making `Edge.value` generic because "you ain't gonna need it. But here we are. Excited to see where this is going. If you wanna try, give a look at `Graph.Edge` in `Graph.swift`. The `value` instance property is currently a `Double`, but we could make `Edge` generic over some `Value` type. We would want to be thoughtful about the generic requirements of the `Value` type (`Hashable`, `Comparable`, `Numeric`, etc).

Reply:
As of Swift 4.2, `Hashable` conformity comes for free (if the composed types are also `Hashable`, which implies (all?) primitive types). Right now, at least the `Node` values are stored in a `Dictionary` (roughly a `HashMap`) as keys. We don't necessarily need it for the `Edge` values, but in case we want `O(1)` access to 'em at some point, it could be worth investigating.

Luckily, it should only have to take adding `: Hashable`, and its there.